socialdistribution/socialdistribution/urls.py

```diff
@@ -31,7 +31,9 @@
     path('new_post/', include('profiles.urls')),
     path('editprofile/', profiles_views.edit_profile, name='editprofile'),
     path('viewprofile/', profiles_views.view_profile, name='viewprofile'),
-
+    path(
+        'author/<uuid:author_id>/profile', profiles_views.view_author_profile, name='view_author_profile'
```

Comment:
What is the difference between `author/<uuid:author_id>/profile` and `/stream/author/<uuid:author_id>/profile`?  Why were both added?

Reply:
Just for testing. removed.